### DISCUSSION TITLE
why are my benefit request are rejected more than 14 times?

### DISCUSSION THREAD

**Discussion by HJDCZY** [community/community#50928] [GitHub Education]
my request has been rejected for more than 14 times! 
![image](https://user-images.githubusercontent.com/112754310/227539431-47f2efd3-670a-4b21-ba39-d1b0e9abc32b.png)
I have already uploaded my Student ID card, OFFICIAL TRANSCRIPT OF UNDERGRADUATESTUDENT, Online verification report of student status of the Ministry of Education and student Certificate, but GitHub reject them all. I uses email end with ".edu.cn", and I do give my browser the correct address and education IP address.
GitHub send me email and said: "You are significantly more likely to be verified if you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio." or "Please upload a different document to demonstrate your current academic status."  I do not know what happened to my bio? I have already updated all I have to let GitHub verify my identify as a student. what do GitHub want?
Do GitHub have Manual review?

**Comment by stijnv18** [comment]
i have the same issue

**Comment by samus-aran** [comment]
Hi @HJDCZY 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

**Reply by HJDCZY** [reply]
thank for your share, I have applied student account successfully!

**Reply by samus-aran** [reply]
Yay @HJDCZY 🎉 

What did you do differently in case others have a similar issue! What offer or area of GitHub Global Campus are you excited to try first? 🥇
